javascript/javascript1/week1/future-age.js

```diff
@@ -0,0 +1,4 @@
+const YearOfBirth=1994;
+let yearFuture=2027;
+let age=yearFuture-YearOfBirth;
+Console.log(`You will be ${age} years old in ${yearFutur}.`);
```

Comment:
Careful when you are console logging there. You shouldn't need to use the uppercase `C`